services/QuillLMS/client/app/bundles/Teacher/components/classrooms/active_classrooms.tsx

```diff
@@ -45,7 +45,7 @@ const bookEmptySrc = `${process.env.CDN_URL}/images/illustrations/book-empty.svg
 const canvasIconSrc = `${process.env.CDN_URL}/images/icons/canvas.svg`
 const cleverIconSrc = `${process.env.CDN_URL}/images/icons/clever.svg`
 const googleClassroomIconSrc = `${process.env.CDN_URL}/images/icons/google-classroom.svg`
-const reorderSrc = `${process.env.CDN_URL}/images/icons/reorder.svg`
+const reorderSrc = `${process.env.CDN_URL}/images/icons/s/reorder.svg`
```

Comment:
`/s` feels kind of cryptic here.  Unless this is a typo, I would use a full word for the folder.

Reply:
that's just what we already have as the subfolder for all of our small icons in `quill-cdn`

Reply:
You can import the `SMALL_ICON_BASE_SRC` from the `Shared` bundle here.